model/src/main/proto/screens.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+enum ScreenName {
+  ACTIVITY_NAME_UNSPECIFIED = 0;
+  SPLASH_ACTIVITY = 1;
+  PROFILE_CHOOSER_ACTIVITY = 2;
+  ADD_PROFILE_ACTIVITY = 3;
+  HOME_ACTIVITY = 4;
+}
+
+message CurrentAppScreen {
```

Comment:
This & the field member should be documented.

Reply:
done.